### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by bgutai**
- [DEMO LINK](https://bgutai.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://bgutai.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,70 @@
...
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover"
+            >
+            laptops & computers
+            </a>
```
Comment (alexander-ignatow) [inline src/index.html:57]: Mind line indentations
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: block;
+  line-height: 60px;
```
Comment (alexander-ignatow) [inline src/style.css:53]: An elegant solution for the element height with vertically centered text! 👍

File: src/index.html (hunk 1 above)
Comment (BudnikOleksii) [reply src/index.html]: Pay attention to comments, let's fix it and move on.

**Review by BudnikOleksii — APPROVED** [review]
Good job👍🏻

**Pull request closed without merging**